downloader/downloadercfg/downloadercfg.go

```diff
@@ -107,3 +97,17 @@ func New(snapDir string, version string, verbosity lg.Level, downloadRate, uploa
 
 	return &Cfg{ClientConfig: torrentConfig, DownloadSlots: downloadSlots}, nil
 }
+
+func getIpv6Enabled() bool {
+	if runtime.GOOS == "linux" {
+		file, err := ioutil.ReadFile("/sys/module/ipv6/parameters/disable")
+		if err != nil {
+			fmt.Printf("Could not read the file due to this %s error \n", err)
```

Comment:
log.Warn probably

Reply:
My bad, copy pasta from my test rig will fix as adding some tests now too.

Reply:
@AskAlexSharov sorted - didn't add tests, value isn't there imo, attempting to spoof runtime.GOOS probably a bad idea.